packages/preview/socialhub-fa/1.0.0/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "socialhub-fa"
+version = "1.0.0"
+repository = "https://github.com/Bi0T1N/typst-socialhub-fa"
+entrypoint = "socialhub-fa.typ"
+authors = ["Nico Neumann (Bi0T1N)"]
+license = "MIT"
+description = "A Typst library for Social Media references with icons based on Font Awesome."
+keywords = ["social", "media", "icons", "Font Awesome"]
+exclude = ["examples/*.pdf"]
```

Comment:
any particular reason for not excluding the whole examples folder? would be simpler.